### ISSUE TITLE
[bug]: SyntaxError: Unterminated closing brace

### ISSUE THREAD

**Issue by T00rk** [hoppscotch/hoppscotch#4022]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Frontend is not loading and Chrome console shows the following error

```javascript
SyntaxError: Unterminated closing brace
    at cv (message-compiler.esm-browser.js:99:19)
    at d (message-compiler.esm-browser.js:223:25)
    at le (message-compiler.esm-browser.js:713:21)
    at Ce (message-compiler.esm-browser.js:808:20)
    at Object.ce [as nextToken] (message-compiler.esm-browser.js:855:16)
    at l (message-compiler.esm-browser.js:942:19)
    at y (message-compiler.esm-browser.js:1080:37)
    at b (message-compiler.esm-browser.js:1128:25)
    at Object.S [as parse] (message-compiler.esm-browser.js:1143:21)
    at dLe (message-compiler.esm-browser.js:1533:24)

runtime-core.esm-bundler.js:226 
```

### Steps to reproduce

1 - Open hoppscotch self hosted instance

### Environment

Production

### Version

Self-hosted

**EDIT:** It also happens on https://hoppscotch.io

**Comment by JoelJacobStephen** [comment]
@T00rk This issue was most probably caused by a misconfigured i18n file. The fix for this has been merged in the latest release build (`2024.3.2`). Can you verify if this issue is resolved in the latest build?

**Comment by T00rk** [comment]
That's strange because I got this error when I set `hoppscotch/hoppscotch-frontend:latest`. Setting `hoppscotch/hoppscotch-frontend:2024.3.2` resolves the issue.
